File: Tarea7U2/src/main/java/com/mycompany/tarea7u2/BankAccount.java

package com.mycompany.tarea7u2;

public class BankAccount {
    private long accountNumber;
    private char tipe;

    public BankAccount(long accountNumber, char tipe) {
        this.accountNumber = accountNumber;
        this.tipe = tipe;
    }

    public void setAccountNumber(long accountNumber) {
        this.accountNumber = accountNumber;
    }

    public void setTipe(char tipe) {
        this.tipe = tipe;
    }

    public String Array(){
        String x = String.format("Account number: " + this.accountNumber + "\nTipe of account: " + tipe + "\n");
        return x;
    }          
}


    

File: Tarea7U2Python/BankAccount.py
class BankAccount:
    def __init__(self, account_number, account_type):
        self.account_number = account_number
        self.account_type = account_type

    def __str__(self):
        return f"Account number: {self.account_number}\nAccount type: {self.account_type}\n"